packages/react-instantsearch/src/components/__snapshots__/Pagination.test.js.snap

```diff
@@ -62,7 +62,7 @@ exports[`Pagination allows toggling display of the first page button on and off
   >
     <a
       aria-label="Page 9"
-      className="ais-Pagination__itemLink"
+      className="ais-Pagination__itemLink ais-Pagination__itemLinkSelected"
```

Comment:
It's due to the fact that I removed the `transitionState` function from the pagination. I need to check for further precision but I believe it comes from the test data context.